documentation/scaling_puppet_server.markdown

```diff
@@ -123,14 +122,12 @@ Before running `puppet agent` or `puppet master` on the new server:
         dns_alt_names = puppet,puppet.example.com,puppet.site-a.example.com
         ```
 
-     If the agent or master has been run and already created a certificate, remove it by running `sudo rm -r $(puppet master --configprint ssldir)`. If an agent has requested a certificate from the master, delete it there to re-issue a new one with the alt names: `puppetserver ca clean master-2.example.com`.
+     If the agent or master has been run and already created a certificate, remove it by running `sudo puppet ssl clean $(puppet master --configprint ssldir)`. If an agent has requested a certificate from the master, delete it there to re-issue a new one with the alt names: `puppetserver ca clean master-2.example.com`.
```

Comment:
Should need to pass it the ssldir when using that command. `puppet ssl clean` should know what to do.